controllers/numaresourcesoperator_controller.go

```diff
@@ -178,15 +179,19 @@ func (r *NUMAResourcesOperatorReconciler) reconcileResource(ctx context.Context,
 	var err error
 	err = r.syncNodeResourceTopologyAPI()
 	if err != nil {
+		r.Recorder.Eventf(instance, corev1.EventTypeWarning, "FailedAPISync", "failed to sync noderesourcetopology api: %v", err)
 		return ctrl.Result{}, status.ConditionDegraded, errors.Wrapf(err, "FailedAPISync")
 	}
+	r.Recorder.Eventf(instance, corev1.EventTypeNormal, "SuccessfulAPISync", "noderesourcetopology synchronization completed")
```

Comment:
this and the above probably deserve a event, but we should revisit the message to make it less low-level.
Something like "Node Resource Topology CRD installed" or so. Need to think a bit about it.